contributors/devel/sig-release/cherry-picks.md

```diff
@@ -62,21 +62,6 @@ your case by supplementing your PR with e.g.,
 - Key stakeholder SIG reviewers/approvers attesting to their confidence in the
   change being a required backport
 
-To illustrate the point, dependency updates that just aim to silence some scanners
```

Comment:
moved this under the security bullet above

Reply:
thanks